src/TestSuite/EmailTrait.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         3.7.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite;
+
+use Cake\TestSuite\Constraint\MailContainsConstraint;
+use Cake\TestSuite\Constraint\MailCountConstraint;
+use Cake\TestSuite\Constraint\MailSentFromConstraint;
+use Cake\TestSuite\Constraint\MailSentToConstraint;
+use Cake\TestSuite\Constraint\MailSentWithConstraint;
+use Cake\TestSuite\Constraint\NoMailSentConstraint;
+
+/**
+ * Make assertions on emails sent through the Cake\TestSuite\TestEmailTransport
+ *
+ * **tests/bootstrap.php**
+ * ```
+ * use Cake\Mailer\Email;
+ * use Cake\TestSuite\TestEmailTransport;
+ *
+ * // replace with other transport configs if required
+ * $config = Email::getConfigTransport('default');
+ * $config['className'] = TestEmailTransport::class;
+ * Email::dropTransport('default');
+ * Email::setConfigTransport('default', $config);
```

Comment:
Is there a way we could make this setup easier to use?  Something like

```php
TestEmailTransport::replaceAllTransports();
```

Reply:
I like that idea!